decred/decred/dcr/account.py

```diff
@@ -4,17 +4,12 @@
 See LICENSE for details
 """
 
-<<<<<<< HEAD
 from decred.crypto import crypto, opcode
-=======
-from decred.crypto import opcode, crypto
->>>>>>> 0ddceba... Check if addr exist in txs
```

Comment:
Sorry about this, isort at a bad time. Should I fix, or just let the final squash erase it?